bouachain/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chain.schema.json",
+  "chain_name": "bouachain",
+  "chain_type": "cosmos",
+  "status": "live",
+  "website": "https://bouachain.com",
+  "network_type": "mainnet",
+  "pretty_name": "Bouachain",
+  "chain_id": "bouachain",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md